interface/usergroup/usergroup_admin_add.php

```diff
@@ -117,6 +117,16 @@ function submitform() {
         }
     } //secure_pwd if ends here
 
+    // Valiate Google email (if provided)
+    if(document.new_user.google_signin_email.value != ""){
+        // https://owasp.org/www-community/OWASP_Validation_Regex_Repository
+        var mailformat = /^[a-zA-Z0-9_+&*-]+(?:\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$/;
+        if(!document.new_user.google_signin_email.value.match(mailformat)) {
+            alert(<?php echo xlj('Google email provided is invalid/not properly formatted (e.g. first.last@gmail.com)') ?>);
```

Comment:
I've never liked this approach of how these alerts are done, but I see you've followed the same pattern as the erx_enable code so we'll keep it. 

I've never been a big believer in regex validators for email, the OWASP isn't technically valid for all RFC 5322 email addresses since it tends to blow up when using local email addresses, however, since this is specific to google email addresses, I'm fine with you using this.

For any other admin's I did confirm this is copied directly from the OWASP example regex validator.